src/generators/app.ts

```diff
@@ -380,13 +381,13 @@ class App extends Generator {
         }
       }
     }
-    if (this.eslint) {
+    if (this.eslint && this.type !== 'sfdx-plugin') {
```

Comment:
Please excuse my ignorance but what does this do?
Does it make it so our standard `eslintrc` isn't added to our own plugins?

Reply:
@RodEsp This repo was a direct clone of the oclif version of plugin generate. The oclif version asked a bunch of question, such as JS or TS, yarn or npm, ... The interactive piece of this plugin was stripped down to be sfdx specific, including language === 'ts', packager mgr === 'yarn', etc. However, all of the logic that consumes the answers were not stripped down, but left in place. Logic like `this.type !== 'sfdx-plugin'` is looking for things to for a sfdx plugin.